train.py

```diff
@@ -65,7 +65,7 @@
         choices=["halving", "median", "none"],
     )
     parser.add_argument("--n-startup-trials", help="Number of trials before using optuna sampler", type=int, default=10)
-    parser.add_argument("--n-evaluations", help="Number of evaluations for hyperparameter optimization", type=int, default=20)
+    parser.add_argument("--n-evaluations", help="Maximum number of training timesteps for each set of hyperparameters during optimization", type=int, default=20)
```

Comment:
same remark as in https://github.com/DLR-RM/rl-baselines3-zoo/pull/102#discussion_r648034503 ...

Reply:
Right sorry. How about "Training policies are evaluated every n-timesteps // n-evaluations steps"?

Reply:
"Training policies are evaluated every n-timesteps // n-evaluations steps when doing hyperparameter optimization"

yes, sounds good ;)